lib/swedbankpay_checkout/client/psp/payment_orders.ex

```diff
@@ -27,16 +27,24 @@ defmodule SwedbankpayCheckout.Client.Psp.PaymentOrders do
   end
 
   @doc """
-  Get a payment order, the payment_order_id should be the full id, with paths, not truncated to the UUID
+  Get a payment order, the payment_order_id should be the full id, with paths, not truncated to the UUID.
+  Expand any subfields listed in the expand parameter.
   """
-  @spec get_payment_order(Tesla.Env.client(), String.t()) ::
+  @spec get_payment_order(Tesla.Env.client(), String.t(), list(Atom.t()) | list(String.t())) ::
```

Comment:
I think we want to take something like a `list(ExpandPaymentOrderField.t())` instead of `list(Atom.t()) | list(String.t())` where `ExpandPaymentOrderField` is defined something like this:
```elixir
defmodule ExpandPaymentOrderField do
  @type t :: :payee | :delivery_address # | etc...
end
``` 

So that we can get some slight amount of discoverability for what is possible to expand. Nice if you could find a better name than ExpandPaymentOrderField too :)